openslides_backend/action/actions/user/set_password_self.py

```diff
@@ -44,12 +45,13 @@ def check_permissions(self, instance: Dict[str, Any]) -> None:
             raise ActionException("Can't set password for anonymous")
 
         instance["id"] = self.user_id
-        if is_temporary(self.datastore, instance):
+        scope, scope_id = self.get_user_scope(self.user_id)
```

Comment:
What is the idea behind this? If the user has Committee or Organisation scope, he's allowed tho change his own password without permission, otherwise he needs the permission?
Or is this meant to call first **check_permissions_for_scope**  and only if it is successfull, check the meeting permission?

Reply:
Perhaps the permissions for committee and orga level are missing. Is there a reason to restrict the users with these levels?

Reply:
@FinnStutzenstein  please answer the questions.

Reply:
The meeting permission `user.can_change_own_password` is a leftover from OS3. There is (currently) no equivalent in committee/orga scope. We have to discuss with Emanuel how this should work there. Until then, there is only a permission check for the meeting to keep the same functionallity to OS3 until we made further decisions.

Reply:
Then we should check it for all meetings of the user, IMO this would be more consistent, although, but not only with OS3

Reply:
Maybe we should leave it as-is since the requirement will change. I try to speak to Emanuel when he is here, so we have a quick (final) decision.

Reply:
Quick documentation of the result: The permission is moved to the user model and can be set orga wide per user, see https://github.com/OpenSlides/OpenSlides/pull/6065